torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -546,12 +583,12 @@ def __init__(self) -> None:
 
     def __call__(
         self,
-        kernel_idx,
-        constant_args_idx,
-        grid,
+        kernel_idx: int,
+        constant_args_idx: int,
+        grid: List["TritonGridType"],
```

Comment:
This is never a callable, right? I thought that Dynamo always applied the grid

Reply:
I ran the tests and the only place where it's a callable is the test `triton_kernel_wrapper_functional_dense`. I'll leave it as is for now and if I get a chance I'll follow-up by changing the test and updating the types?

https://github.com/pytorch/pytorch/blob/a3172809a14f4927bea4aadc0376b5ae8c0d348a/test/inductor/test_triton_kernels.py#L97-L108